docs/components/choice-tiles/test.html

```diff
@@ -86,40 +86,40 @@ <h2>Default</h2>
       <!-- input -->
       <label class="hxChoice">
         <input type="radio" name="default1" />
-        <hx-tile>
+        <div class="hxTile">
           <hx-icon type="checkmark"></hx-icon>
           <div class="hx-tile-icon">
             <hx-icon type="account"></hx-icon>
           </div>
-          <hx-tile-title>Default</hx-tile-title>
-          <hx-tile-description>With a short description or help text.</hx-tile-description>
-        </hx-tile>
+          <header>Default</header>
+          <p>With a short description or help text.</p>
+        </div>
       </label>
 
       <!-- input + :hover -->
       <label class="hxChoice">
         <input type="radio" name="default3" />
-        <hx-tile class="pseudo-hover">
+        <div class="hxTile pseudo-hover">
           <hx-icon type="checkmark"></hx-icon>
           <div class="hx-tile-icon">
             <hx-icon type="account"></hx-icon>
           </div>
-          <hx-tile-title>Default (Hover)</hx-tile-title>
-          <hx-tile-description>With a short description or help text.</hx-tile-description>
-        </hx-tile>
+          <header>Default (Hover)</header>
+          <p>With a short description or help text.</p>
+        </div>
       </label>
 
       <!-- input:focus -->
       <label class="hxChoice">
         <input type="radio" name="default2" class="pseudo-focus" />
-        <hx-tile>
+        <div class="hxTile">
```

Comment:
Check if `.pseudo-focus` should be on the tile like hover

Reply:
the input gets the `:focus` so when a user tab to a tile, the `input` is focused, not the div, double checked with @CITguy on this one

Reply:
Yup, focus applied to input, not div. Marking comments resolved.